securethenews/requirements.in

```diff
@@ -1,4 +1,5 @@
 https://github.com/freedomofpress/django-logging/zipball/34fbeeea7a83bd54f8551bb50382a06b69814d4e
+coverage
```

Comment:
I'm wondering if it's necessary for this dependency to be in the production requirements file, or if it could be moved to `dev-requirements.in` without much change to the code or functionality? It just seems to me like doing the coverage analysis is something we only need when developing or in CI, but I can understand if there's something in the implementation that requires (or is easier if) the package is installed in the prod environment.